src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/BlackDuckMessageBuilder.java

```diff
@@ -33,16 +33,26 @@
 import com.synopsys.integration.blackduck.service.bucket.BlackDuckBucket;
 import com.synopsys.integration.exception.IntegrationException;
 
-public interface BlackDuckMessageBuilder<T> {
-    default String getProviderName() {
-        return "Black Duck";
+public abstract class BlackDuckMessageBuilder<T> {
+
+    private final String providerName = "Black Duck";
```

Comment:
Our integration team convention is to not leave whitespace between the class-name and the first variable/method.